core/http-auth-aws/src/main/java/software/amazon/awssdk/http/auth/aws/signer/CredentialScope.java

```diff
@@ -13,19 +13,19 @@
  * permissions and limitations under the License.
  */
 
-package software.amazon.awssdk.http.auth.internal.util;
+package software.amazon.awssdk.http.auth.aws.signer;
 
 
-import static software.amazon.awssdk.http.auth.internal.util.SignerConstant.AWS4_TERMINATOR;
-import static software.amazon.awssdk.http.auth.internal.util.SignerUtils.formatDate;
-import static software.amazon.awssdk.http.auth.internal.util.SignerUtils.formatDateTime;
+import static software.amazon.awssdk.http.auth.aws.util.SignerConstant.AWS4_TERMINATOR;
+import static software.amazon.awssdk.http.auth.aws.util.SignerUtils.formatDate;
+import static software.amazon.awssdk.http.auth.aws.util.SignerUtils.formatDateTime;
 
 import java.time.Instant;
 import software.amazon.awssdk.annotations.Immutable;
-import software.amazon.awssdk.annotations.SdkInternalApi;
+import software.amazon.awssdk.annotations.SdkProtectedApi;
 import software.amazon.awssdk.identity.spi.AwsCredentialsIdentity;
 
-@SdkInternalApi
+@SdkProtectedApi
 @Immutable
 public final class CredentialScope {
```

Comment:
Any way to avoid this needing to be protected?

Reply:
It's used where an impementation of the base-v4-signer is defined, so no.

Reply:
Okay, let's put this on the back burner for now, then. I think I'll start taking passes at the protected and public APIs to make sure they're buttoned up so there's not a lot of controversy during the surface area review.